mkosi/__init__.py

```diff
@@ -5372,10 +5370,11 @@ def create_parser() -> ArgumentParserMkosi:
     group.add_argument("--image-version", help="Set version for image")
     group.add_argument("--image-id", help="Set ID for image")
     group.add_argument(
-        "--no-chown",
+        "--chown",
```

Comment:
I'm fine with this change in principal and we should have done this like this from the start. Could you add `--no-chown`, though as a suppressed option that goes into the same dest? `BooleanAction` does a lot (`--not-chown`, `--withouth-chown`), but I think with this change passing `--no-chown` would be an error.

Reply:
I tend to agree but on the other hand, we're going to be breaking compat all over the place anyway in this release, so maybe we should take the time and get rid of all the deprecated stuff and do some extra breaking changes without caring for backwards compat?